sdk/servicebus/service-bus/test/streamingReceiver.spec.ts

```diff
@@ -867,22 +872,18 @@ describe("Streaming - Failed init should not cache recevier", function(): void {
     await afterEachTest();
   });
 
-  class TestTokenProvider extends SasTokenProvider {
+  class TestTokenProvider extends SharedKeyCredential {
     private firstCall = true;
```

Comment:
Work was done here previously for enabling the test when updating to use `core-amqp` before the base branch was upsated.
Did we not want to use `SharedKeyCredential` ? It doesn't seem to work in browser too, but the ones defined in `@azure/identity` such as `EnvironmentCredential` as well are not browser tests compatible yet I think. Which one did we want to use now as such? @ramya-rao-a

Reply:
`SharedKeyCredential` is currently meant for internal use only i.e the Service Bus and the Event Hubs packages will use them internally when user passes a connection string.

Users are expected to use the credentials from `@azure/identity` or create their own by implementing the `TokenCredential` interface.